blog_system/settings.py

```diff
@@ -0,0 +1,144 @@
+"""
+Django settings for blog_system project.
+
+Generated by 'django-admin startproject' using Django 4.1.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.1/ref/settings/
+"""
+
+from pathlib import Path
+import os
+from dotenv import load_dotenv
+
+
+load_dotenv()
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = os.getenv("VARABLE", "dummy-key")
+
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = "True"

```

Comment:
The `DEBUG` setting is currently set to the string 'True'. It should be a boolean value `True` or `False`. Consider changing it to `DEBUG = True` for development purposes, but remember to set it to `False` in production.